satpy/readers/pmw_channels_definitions.py

```diff
@@ -0,0 +1,411 @@
+        return f"central={self.central} {self.unit} ±{self.side} ±{self.sideside} width={self.bandwidth} {self.unit}"
+
+    def __hash__(self):
+        """Hash this tuple."""
+        return tuple.__hash__(self)
+
+    def __contains__(self, other):
+        """Check if this quadruple-side-band 'contains' *other*."""
+        if other is None:
+            return False
+
+        # The four centrals:
+        central_left_left = self.central - self.side - self.sideside
+        central_left_right = self.central - self.side + self.sideside
+        central_right_left = self.central + self.side - self.sideside
+        central_right_right = self.central + self.side + self.sideside
+
+        four_centrals = [central_left_left, central_left_right,
+                         central_right_left, central_right_right]
+        if isinstance(other, numbers.Number):
+            for central in four_centrals:
+                if _is_inside_interval(other, central, self.bandwidth):
+                    return True
+
+            return False
+
+        if isinstance(other, (tuple, list)) and len(other) == 5:
+            raise NotImplementedError("Can't check if one frequency quadruple side band is contained in another.")
+
+        with suppress(AttributeError):
+            if self.unit != other.unit:
+                raise NotImplementedError("Can't compare frequency ranges with different units.")
+
+        return False
+
+    def distance(self, value):
+        """Get the distance to the quadruple side band.
+
+        Determining the distance in frequency space between two quadruple side
+        bands can be quite ambiguous, as such bands are in effect a set of 4
+        narrow bands, two on each side of the main absorption band, and on each
+        side, one on each side of the secondary absorption lines. To keep it as
+        simple as possible we have until further decided to define the distance
+        between such two bands to infinity if they are determined to be equal.
+
+        If the frequency entered is a single value, the distance will be the
+        minimum of the distances to the two outermost sides of the quadruple
+        side band.
+
+        If the frequency entered is a tuple or list and the two quadruple
+        frequency bands are contained in each other (equal) the distance will
+        always be zero.
+
+        """
+        left_left = self.central - self.side - self.sideside
+        right_right = self.central + self.side + self.sideside
+
+        if self == value:
+            try:
+                left_side_dist = abs(value.central - value.side - value.sideside - left_left)
+                right_side_dist = abs(value.central + value.side + value.sideside - right_right)
+            except AttributeError:
+                if isinstance(value, (tuple, list)):
+                    msg = 'Distance to a quadruple side band frequency not supported for this type'
+                    raise NotImplementedError(msg)
```

Comment:
Is this tested?

Reply:
You got me! Right, it wasn't. It is now. But the `NotImplementedError` would never happen as it is implemented right now. So, I removed the check for tuple/list.